docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -226,6 +226,86 @@ defs = Definitions(
 )
 ```
 
+### Checks on partitioned assets
+
+To write a check on a partitioned asset, summarize the status of all the partitions as a single pass/fail for the asset.
+
+Here's an example that checks that no partitions are null.
```

Comment:
Thinking about an asset from the perspective of a file stored on disk or a table in a database, it's not clear what it would mean for a partition to be "null". Something like verifying that all of the partitions have more than one row (or more than one byte?) would be more relatable.

Reply:
how about all partition values are greater than 0

Reply:
What would 'value' here mean, though? Should we expect the reader to know what that means?

Reply:
Yeah I think a partition normally includes a chunk of data, wouldn't typically include a single value.